### PR TITLE
“Horizontal scaling” zh trans

### PR DESCRIPTION
**Pull request by ethan-daocloud**
"水平扩展" for "Horizontal scaling", it's more meaningfull and understandable

### CODE REVIEW

File: content/zh/_index.html [hunk 1]
```diff
@@ -66,7 +66,7 @@ <h4><a href="/docs/concepts/workloads/controllers/replicationcontroller/#what-is
         <div class="feature-box">
             <div>
                 
-                <h4><a href="/docs/tasks/run-application/horizontal-pod-autoscale/">横向缩放</a></h4>
```
Comment (chenrui333) [inline content/zh/_index.html]: Actually, this is a translation from the doc, 
https://github.com/k8smeetup/k8s-official-translation/blob/master/trans-glossary.md#glossary-for-translation

I actually prefer `横向缩放` but the two are pretty much the same though.

Comment (ethan-daocloud) [reply content/zh/_index.html]: good point. I got it.

Comment (tengqm) [reply content/zh/_index.html]: agree to @chenrui333

**Pull request closed without merging**